iterations.r

```diff
@@ -61,9 +62,30 @@ stations_metadata_df %>%
   GQL(., .url = configs$vegvesen_url) %>%
   transform_volumes() %>% 
   ggplot(aes(x=from, y=volume)) + 
-  geom_line() + 
+  geom_line() +
   theme_classic()
 
+### 6: making the plot prettier
+#Select a random station and saving it in a variable
+selected_station <-
+  stations_metadata_df %>% 
+  filter(latestData > Sys.Date() - days(7)) %>% 
+  sample_n(1)
+
+#Create volume plot
+selected_station %$% 
+  vol_qry(
+    id = id,
+    from = to_iso8601(latestData, -4),
+    to = to_iso8601(latestData, 0)
+  ) %>% 
+  GQL(., .url = configs$vegvesen_url) %>%
+  transform_volumes() %>% 
+  ggplot(aes(x=from, y=volume)) + 
+  geom_line() +
+  theme_classic() +
+  ggtitle(paste("Traffic volume plot at:",selected_station$name))
+  
```

Comment:
Very nice plot! Consider making the x and y-axis labels a bit nicer. E.g making the first letter uppercase and perhaps write something more descriptive than "from".